src/foxops/models/incarnation.py

```diff
@@ -34,7 +33,6 @@ class Config:
 
 
 class IncarnationWithDetails(IncarnationBasic):
-    status: ReconciliationStatus = Field(description="DEPRECATED. Use the 'merge_request_status' field instead.")
```

Comment:
@umer4ik 
I briefly searched the UI code to check if there are still any references to this field. I couldn't find any - but could you maybe quickly double check?